packages/next/src/compiled/react-dom/cjs/react-dom-server.edge.development.js

```diff
@@ -8805,5 +8805,7 @@
         startWork(request$jscomp$0);
       });
     };
-    exports.version = "19.0.0-rc-187dd6a7-20240806";
+    
+;const setTimeoutOrImmediate = typeof setImmediate === 'function' ? setImmediate : setTimeout;
```

Comment:
i don't know if this matters, but we might want to move this to the very end, like
```
  exports.version = "..."; const setTimeoutOrImmediate = ...
```
i think i've seen this done that way to mess with sourcemaps as little as possible?

Reply:
Right, I guess i forgot there's no sourcemaps here 🤦‍♀️